specs/altair/beacon-chain.md

```diff
@@ -26,6 +26,8 @@
     - [`SyncAggregate`](#syncaggregate)
     - [`SyncCommittee`](#synccommittee)
 - [Helper functions](#helper-functions)
+  - [Crypto](#crypto)
+    - [BLS public keys](#bls-public-keys)
   - [`Predicates`](#predicates)
     - [`eth2_fast_aggregate_verify`](#eth2_fast_aggregate_verify)
```

Comment:
It was okay to have `eth2_fast_aggregate_verify` in the "Predicates" category, but since this PR adds `eth2_aggregate_pubkeys`, I think it makes sense to put them together. Like:
```suggestion
  - [Crypto](#crypto)
      - [BLS signatures](#bls-signatures)
          - [`eth2_fast_aggregate_verify`](#eth2_fast_aggregate_verify)
          - [`eth2_aggregate_pubkeys`](#eth2_aggregate_pubkeys)
```